content/en/docs/ops/deployment/vm-architecture/index.md

```diff
@@ -0,0 +1,133 @@
+
+Control plane traffic, including XDS configuration and certificate signing, are sent through a Gateway in the cluster.
+Similarly, all communication between pods and virtual machines goes through the gateway, which acts as a bridge between the two networks.
+
+{{< image width="75%"
+    link="multi-network.svg"
+    alt="A service mesh with multiple networks and virtual machines"
+    title="Multiple networks"
+    caption="A service mesh with multiple networks and virtual machines"
+    >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
+
+## Service association
+
+Istio introduces two new types to represent virtual machine workloads:
+
+* [`WorkloadGroup`](/docs/reference/config/networking/workload-group/) represent a grouping of workloads that share common properties. This is comparable to a `Deployment` in Kubernetes
+* [`WorkloadEntry`](/docs/reference/config/networking/workload-entry/) represent a single instance of a workload. This is comparable to a `Pod` in Kubernetes.
+
+Unlike their corresponding Kubernetes resources, creating these resources does not result in provisioning of any resources or running any applications.
+Rather, these resources just reference these workloads and inform Istio how to configure the mesh appropriately.
+
+When adding a virtual machine workload to the mesh, you will need to create a `WorkloadGroup`, that acts as template for each instance:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: WorkloadGroup
+metadata:
+  name: product-vm
+spec:
+  metadata:
+    labels:
+      app: product
+  template:
+    serviceAccount: default
+  probe:
+    httpGet:
+      port: 8080
+{{< /text >}}
+
+Once the virtual machine has been [configured and added to the mesh](/docs/setup/install/virtual-machine/#configure-the-virtual-machine), a corresponding `WorkloadEntry` will be automatically created.
+For example:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1beta1
+kind: WorkloadEntry
+metadata:
+  annotations:
+    istio.io/autoRegistrationGroup: product-vm
+  labels:
+    app: product
+  name: product-vm-1.2.3.4
+spec:
+  address: 1.2.3.4
+  labels:
+    app: product
+  serviceAccount: default
+{{< /text >}}
+
+These `WorkloadEntry` describe a single instance of a workload, similar to a pod. When the workload is removed from the mesh, the `WorkloadEntry` will
+be automatically removed, and its health status will be automatically updated based on the probe configured in the `WorkloadGroup`, if any.
+
+In order for consumers to actually call the workload, generally it's desired to join a part of a `Service`. This allows
+clients to reach a stable hostname, like `product.default.svc.cluster.local`, rather than an IP address that may change.
+Additionally, it allows Istio configuration such as `DestinationRule`s and `VirtualService`s to apply.
```

Comment:
```suggestion
In order for consumers to reliably call your workload, it's recommended to declare a `Service` association. This allows clients to reach a stable hostname, like `product.default.svc.cluster.local`, rather than ephemeral IP addresses. This also enables you to use advanced routing capabilities in Istio via `DestinationRule` and `VirtualService` APIs.
```